packages/openapi-v3/README.md

```diff
@@ -0,0 +1,87 @@
+@loopback/openapi-v3
+
+This package contains:
+
+* Decorators that describe LoopBack artifacts as OpenAPI 3.0.0 metadata.
+* Utilities that transfer LoopBack metadata to OpenAPI 3.0.0 specifications.
+
+## Overview
+
+The package has functions described above for LoopBack controller classes.
+Decorators apply REST api mapping metadata to controller classes and their members. And utilities that inspect controller classes to build OpenAPI 3.0.0 specifications from REST api mapping metadata.
+
+Functions for more artifacts will be added when we need.
+
+## Installation
+
+```
+$ npm install --save @loopback/openapi-v3
+```
+
+## Basic use
+
+Currently this package only has spec generator for controllers.
+It generates swagger `paths` and `basePath` specs for a given decorated controller class.
```

Comment:
I suggest to re-phrase this sentence as `It generates OpenAPI 'paths' and 'servers' specs for...`